SPECS/kernel/kernel.spec

```diff
@@ -2,6 +2,12 @@
 %global sha512hmac bash %{_sourcedir}/sha512hmac-openssl.sh
 %define uname_r %{version}-%{release}
 
+# Edit default debuginfo settings
```

Comment:
for reference, fedora's corresponding section for these changes:
https://src.fedoraproject.org/rpms/kernel/blob/rawhide/f/kernel.spec#_671

shouldn't these be protected inside %if %{with_debuginfo} like fedora does? If not, wouldn't these changes make it impossible for us to build a kernel rpm without debuginfo (i.e. rpmbuild --without debuginfo would now fail, I think)

also as you noted, some of the other settings there might be useful and/or needed before merging this? At least a quick investigation for each param?

Reply:
Mariner does not support these macros (%with_debuginfo) as of today. I did a quick test and running `rpmbuild -ba --nodebuginfo /usr/src/mariner/SPECS/kernel.spec 2>&1 | tee kernel-defconfig.log` did not fail due to the new macros but it DID fail because vmlinux is now unpackaged file. Fedora also protects the vmlinux install step: https://src.fedoraproject.org/rpms/kernel/blob/rawhide/f/kernel.spec#_2160

Reply:
ah ok, so our build already didn't work without debuginfo, so this doesn't break anything that wasn't already broken. It doesn't seem like anyone was doing any non-debuginfo builds (or else someone presumably would have fixed it already) so it LGTM. Thanks!